packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -183,13 +184,14 @@ export default class QuanticCaseClassification extends LightningElement {
           Math.max(Number(this.maxSuggestions), 0)
         ) ?? [];
       this.loading = this.field.state.loading;
```

Comment:
The loading spinner still appears even though the value is fixed and won't update. Seems like an unecessary and potentially confusing visual feedback. What do you think about altering the logic so it doesn't appear?

Reply:
Absolutely 👌 I'll fix that

Reply:
Fixed👌